Automattic/jetpack
# Boost: Update deferjs to exclude application/json

Fixes #19158

#### Changes proposed in this Pull Request:

* Adds `application/json` script type to the exclusion logic while deferring Javascript.

#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Activate Jetpack
* Activate Boost and its "Defer Non-Essential JavaScript" option
* Go to Posts > Add New and add a new Podcast block
* Paste in a podcast URL such as https://distributed.blog/category/podcast/feed/
* Publish your post
* Ensure that the player works well in the editor as well as in the front end.
